craco.config.js

```diff
@@ -6,7 +6,9 @@ const path = require('path')
 module.exports = function() {
   return {
     webpack: {
-      alias: {},
+      alias: {
+        '@': path.resolve(__dirname, 'src/custom')
```

Comment:
well I was referring to use an alias for their src:

```js
        '@': path.resolve(__dirname, 'src/custom')
```

Just so we don't need to use  relative paths (../../.../) in our custom overrides of components